### PR TITLE
Resolves #10, #11 

### LINKED ISSUE #10: Save page urls brought from Feedly to the DB
@leesera

### PR DESCRIPTION
**Pull request by leesera**
- Show the list of page which is from DB
- Put the url from feedly account at DB

### CODE REVIEW

File: Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java [hunk 1]
```diff
@@ -28,6 +29,7 @@
   static final String ACCESS_TOKEN = "A06EprS0187tNdGMJ1XPTVQa1eE8SeGLXZeK3GZy2UwZ8qzOGSqZlPmXNcYul0zueeQRLYwN1nWbFszj6PyoNOkCGSbUp9zfJ3eLROo3bJWsUQktkXPfbFruJn9TGFQQ5r16aLhP7f-VXMFNxMtlrJw21eabhWzhzO-9r0OkXBesU_0Kscpb4SaRPW4TpYpfGiusnAKhaWmeNYdu5VaCGMdFpoch:feedlydev";
   static final String ID = "3d0c7dd1-a7bb-4cdf-92f0-6c25d88c52db";
 
+
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java]: Meaningless empty line

File: Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java [hunk 2]
```diff
@@ -55,11 +57,18 @@ protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_main);
 
+    final Context mainActivity = this;
+
     Button btnFeedlyAccount = (Button) findViewById(R.id.FeedlyAccountBtn);
     btnFeedlyAccount.setOnClickListener(new Button.OnClickListener() {
       public void onClick(View v) {
+
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java]: ditto

File: Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java [hunk 3]
```diff
@@ -55,11 +57,18 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        DataBaseOpenHelper dataBaseOpenHelper;
+        dataBaseOpenHelper = new DataBaseOpenHelper(mainActivity);
+
         final String URL = "https://cloud.feedly.com/v3/streams/contents?streamId=user/" + ID + "/category/global.all";
-        new GetPageList().execute(URL);
+        new GetPageList(dataBaseOpenHelper, mainActivity).execute(URL);
+
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java]: ditto

File: Roys/app/src/main/java/com/dwg_karrier/roys/ListActivity.java [hunk 4]
```diff
@@ -47,24 +47,17 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
...
-    wholeList.add(new ScriptedURL("temp url2", 1, 13));
-    wholeList.add(new ScriptedURL("temp url3", 1, 1123));
-    wholeList.add(new ScriptedURL("temp url4", 1, 13921));
-    wholeList.add(new ScriptedURL("temp url5", 1, 23213));
 
     for (ScriptedURL temp : wholeList) {
       tempTime = (double) temp.getWordCount() / wordsperMin;
       Log.d("test tempTime", "" + tempTime + "wordCount" + temp.getWordCount());
-      ret.add(new ScriptedData("", tempTime, ""));
+      ret.add(new ScriptedData("asdf", tempTime, "ada"));
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/ListActivity.java]: What does this line mean?

File: Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java (hunk 1 above)
Comment (leesera) [inline Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java]: ok!

File: Roys/app/src/main/java/com/dwg_karrier/roys/ListActivity.java (hunk 4 above)
Comment (leesera) [inline Roys/app/src/main/java/com/dwg_karrier/roys/ListActivity.java]: It is just test code:) I will remove it

File: Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java [hunk 5]
```diff
@@ -87,15 +87,43 @@ public void setWordCount(String url, int wordCount) {
...
     dataBase.close();
   }
 
+  //For test
+  public void getTableAsString() {
+    SQLiteDatabase db = getReadableDatabase();
+    String tableName = "page";
+
+    String tableString = String.format("Table %s:\n", tableName);
+    Cursor allRows  = db.rawQuery("SELECT * FROM " + tableName, null);
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java]: `all Rows = `

File: Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java [hunk 6]
```diff
@@ -87,15 +87,43 @@ public void setWordCount(String url, int wordCount) {
...
+    if (allRows.moveToFirst() ){
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java]: `moveToFirst()) {`

**Comment by leesera** [comment]
I fixxxxxed it!!

**Comment by xhae** [comment]
Thank you so much!

**Pull request merged**